components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -881,9 +880,10 @@ void ConversationDriver::SubmitSelectedTextWithQuestion(
                 return;
               }
 
-              received_callback.Run(suggestion);
+              accumulator += suggestion;
```

Comment:
**Note:** This mostly works but the string is quoted which feels slightly weird

@petemill @evq